src/ContentBundle/Block/Service/JumbotronBlockService.php

```diff
@@ -55,7 +55,7 @@ public function buildManageForm(FormBuilderInterface $builder, array $options)
             $builder->get('properties')
                 ->add('styles', ChoiceType::class, [
                     'label' => 'label.styling',
-                    'choices'  => $this->config['styles'],
+                    'choices'  => array_flip($this->config['styles']),
```

Comment:
I don't agree with this change. The config itself should be updated, not the way the config is read in the service.